Sources/Java2SwiftLib/JavaTranslator.swift

```diff
@@ -251,7 +251,12 @@ extension JavaTranslator {
         $0.flatMap { field in
           if field.isStatic {
             staticFields.append(field)
-            return nil
+
+            if field.isEnumConstant() {
+              return "public static let \(raw: field.getName()) = try! JavaClass<Self>(in: JavaVirtualMachine.shared().environment()).\(raw: field.getName())!"
```

Comment:
If we've going to use `JavaVirtualMachine` here, we need to also make sure to `import JavaKitVM` in the generated code. There's a set of imports in the `JavaTranslator` that you would need to add to.

I do worry a bit about calling `JavaVirtualMachine.shared()` here in something that looks like a constant, and I wonder if we should consider alternative ways to model Java enums.